src/frida.d.ts

```diff
@@ -1,337 +0,0 @@
-// This file is based on http://www.frida.re/docs/javascript-api
```

Comment:
These types are contained in an NPM types package so we can include them in package.json and not ship them with the source code.